draft-scenarios/clash/arcane_artillery.tex

```diff
@@ -0,0 +1,141 @@
+\end{itemize}
+
+\textbf{For a 3-6 player Scenario:}
+\begin{itemize}
+  \item P × Starting (I) Map Tile
+  \item 6 × Near (IV--V) Map Tile that contains an Obelisk. If not enough are available, treat Witch Huts as Obelisks
+  \item 1 × Center (VI--VII) Map Tile, which must contain the Dragon Utopia Field
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+To win the Scenario, a Hero must Flag the Dragon Utopia Field.
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+At the end of the \nth{12} Round, if there is no winner, all players lose the Scenario.
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\textbf{\nth{6} Round:}
+\begin{itemize}
+  \item Remove all Black cubes from every Tile.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+
+During this Scenario:
+
+\begin{itemize}
+  \item Heroes cannot enter starting Tiles other than their own.
+  \item A Player may not use the Diplomacy card to skip Combat on the Dragon Utopia Field.
+  \item Arcane Spikes are represented by cubes not used by any player's Faction.
+  \item Obelisks may be Flagged. 
+  \item Each Obelisk can only be Flagged by one Player at a time.
+  \item When first Flagged, that Player gains one Arcane Spike.
+  \item At the start of each Player's turn, that Player removes any unspent Arcane Spikes, then gains Arcane Spikes equal to the number of captured Obelisks they have.
+\end{itemize}
+
+\vspace{1em}
+
+\includegraphics[width=\linewidth, keepaspectratio]{\art/prayer.png}
+
+\vspace*{\fill}
+
+\columnbreak
+
+\textbf{Players can spend one Arcane Spike during combat to cast:}
+
+\hommtable[]{15}{
+  \begin{tabularx}{\linewidth}{X}
+    \darkcell{\raisebox{-.2\height}{} \textbf{Arcane Artillery}}\\
+    \lightcell[1.8]{\raisebox{-.25\height}{\svg{activation_white}} Deal 1 \svg{damage-table} to target Unit by placing an Arcane Spike on it.} \\ 
+    \lightcell[3.8]{\raisebox{-.25\height}{\svg{ongoing_white}} A Unit with an Arcane Spike placed on it has its special ability negated \\ and receives +1 \svg{damage-table} from every source of \\ \svg{damage-table} and attacks for every Arcane Spike placed on them (even if they were to receive 0 damage)} \\
+  \end{tabularx}
+}
+
+\textbf{Additionally:}
+
+\begin{itemize}
+  \item Arcane Artillery ignores Azure Dragon's Special Ability when cast during \textbf{Level VII} combat.
+  \item Arcane Artillery does not count toward your combat Round Spell limit.
+  \item Arcane Artillery is considered to be a Spell of every school of magic of \svg{empower} 0.
+  \item Players can cast Srcane Srtillery during combat involving opponent's Heroes \textbf{during the neutral Unit's activation}.
+  \item Each player can cast only \textbf{one Arcane Artillery per combat Round}.
+  \item \textit{In alliance mode, allies can cast Arcane Artillery on allied Unit's activation.}
+\end{itemize}
+
+\end{multicols}
+
+\newpage
+
+  \begin{minipage}{0.4\paperwidth}
```

Comment:
In general, I'd discourage using minipages for this. IMO, tikzpicture works much better, because it's easier to move things around and keep pictures to scale. On that note, `\linewidth` works better, because the former is pinned to the available space, and the latter to the width of the whole page, which in this case makes little sense. But, I guess, in this scenario we can leave it as it is.

Reply:
Sure. I was lazily copy pasting from the Titans Stronghold scenario since it had a similar amount of maps, but I can use tikzpicture next time